testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/crossdc/SessionExpirationCrossDCTest.java

```diff
@@ -72,6 +73,18 @@
     private int authSessions02;
 
 
+    @Override
```

Comment:
TO be removed: It was an attempt to fix the crossdc-server tests without success